Core/NoOpRunner.Core/NoOpRunner.cs

```diff
@@ -77,6 +77,7 @@ public void StartHosting()
 
         public void FireLoop()
         {
+            Player.OnLoopFired((WindowPixel[,])GameWindow.GetCurrentWindow().Clone());
```

Comment:
Why not reuse the same result from .GetCurrentWindow?

Reply:
Yes